docs/user/reference/charm/index.md

```diff
@@ -1,97 +1,242 @@
 (charm)=
 # Charm
 
+> See also: {ref}`manage-charms`
+
 ```{toctree}
 :hidden:
-
-channel
-revision
-charm-taxonomy
 charm-development-best-practices
 charm-naming-guidelines
 charm-maturity
-charm-environment-variables
 ```
 
+In Juju, a **charm** is an operator -- software that wraps an {ref}`application <application>` and that contains all of the instructions necessary for deploying, configuring, scaling, integrating, etc., the applicationon any {ref}`Juju-supported cloud <list-of-supported-clouds`.
+
+
+## The charm store
```

Comment:
"charm store" is confusing to me. it is the place where charms are stored, or the literal CharmStore, precursor to CharmHub?

Reply:
Removed that section title and improved the language.